.github/workflows/python-tests.yaml

```
@@ -321,6 +321,142 @@ jobs:
           && docker container logs postgres
           || echo "Ignoring bad exit code"
 
+  run-pydantic-v2-compat-tests:
+    runs-on:
+      group: oss-larger-runners
+    name: pydantic v2 compat, python:${{ matrix.python-version }}
```

```suggestion
  run-pydantic-v2-tests:
    runs-on:
      group: oss-larger-runners
    name: pydantic v2, python:${{ matrix.python-version }}
```
any reason not to just call this v2 since it's going to install and use v2?